test/e2e/framework/framework.go

```diff
@@ -105,26 +110,76 @@ func newFramework(kubeconfig string, qps float32, burst int) (*Framework, error)
 	}, nil
 }
 
+const (
+	kubeconfigFlag      = "kubeconfig"
+	gsimageFlag         = "gameserver-image"
+	pullSecretFlag      = "pullsecret"
+	stressTestLevelFlag = "stress"
+	perfOutputDirFlag   = "perf-output"
+	versionFlag         = "version"
+	featureGatesFlag    = "feature-gates"
+)
+
+// SplitFlags This workaround allows to use flag.Parse() in m.Run and pflag.Parse() together
+// https://github.com/spf13/pflag/issues/63
+func SplitFlags() error {
```

Comment:
I'm not sure I understand why this is here, I couldn't find anywhere else where we use the `flag` library other than in this file? And you've now removed all of that, so do we still need this?

Reply:
Yes, we do need this because of:
![image](https://user-images.githubusercontent.com/16540942/78994529-fe263500-7b48-11ea-9cdd-5a038e64d618.png)

Otherwise, I get an error like _"flag provided but not defined"_.
We do not use viper & pflag in tests that is why there is no such an issue in other places. I didn't find another solution except this.

Reply:
All kind of `go test -v -run TestOne` should be passed to m.Run() unchanged. Them are prefixed with `test.` and could be ignored by `pflag` but needed for test tool. Without this or similar workaround we could not use pflag with testing packages.

Reply:
oooh! It gets called in the Go Test code? I would never have thought of that.

This issue was from 2016, I take there's been no fix since then?

Reply:
https://github.com/spf13/pflag#supporting-go-flags-when-using-pflag seems relevent?

Reply:
Just tried it - works perfectly.

Reply:
Also seen this function before, but this readme section better explains how to use it.

Reply:
Well it seems that KeepGoFlags feature are not yet in place, so this workaround is needed. 
https://github.com/spf13/pflag/issues/63#issuecomment-215143145
And all `go test` flags are ignored by `pflag` lib. @akremsa  has just tested that `-run`, `-v` are skipped.

Reply:
There is a more recent bug with `-test.v` flag
https://github.com/spf13/pflag/issues/238

Reply:
Yes, a bit tricky issue. We fixed passing custom flags by adding `pflag.CommandLine.AddGoFlagSet(flag.CommandLine)`, but broke passing golang test flags.
https://github.com/spf13/pflag/issues/238

I suggest to revert changes back and leave SplitFlags() function.